flight/targets/revolution/board-info/pios_board.h

```diff
@@ -86,10 +86,11 @@ TIM8  |           |           |           |
 
 
 //------------------------
-// PIOS_LED
+// PIOS_ANNUNC
 //------------------------
 #define PIOS_LED_HEARTBEAT	0
 #define PIOS_LED_ALARM		1
+#define PIOS_ANNUNCIATOR_BUZZER	2
```

Comment:
Alignment broken by this

Reply:
Not in reality just in diff (extra + at the beginning throws it off)

Reply:
Oh yeah. Should really use spaces for aligning stuff anyway (but still tabs for indentation ofc).